src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheckTest.java

```diff
@@ -72,6 +72,24 @@ public void testMissingSwitchDefaultSwitchExpressions() throws Exception {
             expected);
     }
 
+    @Test
+    public void testInputMissingSwitchDefaultCheckNullLabel() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verifyWithInlineConfigParser(
+                getNonCompilablePath("InputMissingSwitchDefaultCheckNullLabel1.java"),
+            expected);
+    }
+
+    @Test
+    public void testInputMissingSwitchDefaultCheckNullLabel1() throws Exception {
```

Comment:
Again, if we can't name test methods/inputs meaningfully, we need to reconsider the content and motivation of them.

Reply:
Changed @nrmancuso